src/component/tags/TagTypeList/TagTypeList.tsx

```diff
@@ -123,10 +129,11 @@ export const TagTypeList = () => {
                 <ListItemText primary={link} secondary={tagType.description} />
                 <PermissionIconButton
                     permission={UPDATE_TAG_TYPE}
+                    // @ts-expect-error
```

Comment:
Can you please expand a bit on why you need this here?

Reply:
Nice catch! Just a pending TODO, but why not fix the `PermissionIconButton` props right away. Should be better now.